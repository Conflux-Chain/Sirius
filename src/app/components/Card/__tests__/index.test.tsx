import React from 'react';
import { render } from '@testing-library/react';
import { HelmetProvider } from 'react-helmet-async';

describe('<Tooltip />', () => {
  it('should match the snapshot', () => {
    const component = <div>Todo. Tooltip</div>;
    expect(component).toMatchSnapshot();
  });
});
