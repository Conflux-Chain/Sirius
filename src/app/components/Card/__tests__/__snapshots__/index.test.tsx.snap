// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card  /> should match snapshot 1`] = `
.c0 .card.sirius-Card {
  box-shadow: 0.8571rem 0.5714rem 1.7143rem -0.8571rem rgba(20,27,50,0.12);
  padding: 0.7143rem;
}

.c0 .card.sirius-Card:hover {
  box-shadow: 0.8571rem 0.5714rem 1.7143rem -0.8571rem rgba(20,27,50,0.12);
}

<div
  class="c0"
>
  <div
    class="jsx-2441804542 card sirius-Card"
  >
    <div
      class="jsx-178616938 content "
    >
      test content
    </div>
  </div>
</div>
`;
